guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -26,5 +26,8 @@ ifdef::managing-hosts[]
 endif::[]
 ifndef::managing-hosts[]
 * To migrate a host from Katello Agent, see {ManagingHostsDocURL}Migrating_Hosts_From_Katello_Agent_to_Remote_Execution_managing-hosts[Migrating from Katello Agent to Remote Execution].
-* To enable pull mode on a new host, continue either with {ManagingHostsDocURL}Creating_a_Host_managing-hosts[Creating a Host] or {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
+* To enable pull mode on a new host, continue with either of the following procedures:
+
+** {ManagingHostsDocURL}Creating_a_Host_managing-hosts[Creating a Host]
+** {ManagingHostsDocURL}Registering_Hosts_managing-hosts[Registering Hosts] in _{ManagingHostsDocTitle}_.
```

Comment:
These aren't xrefs, the xrefs are on [line #25](https://github.com/theforeman/foreman-documentation/pull/2440/files#diff-cde150ae9a7e5a21bc485a142ebd7e3b299781b4910cc3c46d951926d7343f4dR25) ^ :)